packages/orchestrator-ui-components/src/rtk/endpoints/subscriptionListSummary.ts

```diff
@@ -0,0 +1,70 @@
+import { CacheTags, orchestratorApi } from '@/rtk';
+import {
+    BaseGraphQlResult,
+    GraphqlQueryVariables,
+    Subscription,
+    SubscriptionSummary,
+    SubscriptionsResult,
+} from '@/types';
+
+export const subscriptionListSummaryQuery = `
+    query SubscriptionsList(
+        $first: Int!
+        $after: Int!
+        $sortBy: [GraphqlSort!]
+        $filterBy: [GraphqlFilter!]
+        $query: String
+    ) {
+        subscriptions(
+            first: $first
+            after: $after
+            sortBy: $sortBy
+            filterBy: $filterBy
+            query: $query
+        ) {
+            page {
+                startDate
+                description
+                subscriptionId
+            }
+            pageInfo {
+                totalItems
+                startCursor
+                endCursor
+            }
+        }
+    }
+`;
+
+export type SubscriptionListSummaryResponse = {
+    subscriptions: SubscriptionSummary[];
+} & BaseGraphQlResult;
+
+const subscriptionListSummaryApi = orchestratorApi.injectEndpoints({
+    endpoints: (builder) => ({
+        getSubscriptionSummaryList: builder.query<
+            SubscriptionListSummaryResponse,
+            GraphqlQueryVariables<Subscription>
+        >({
+            query: (variables) => ({
+                document: subscriptionListSummaryQuery,
+                variables,
+            }),
+            transformResponse: (
+                response: SubscriptionsResult<SubscriptionSummary>,
+            ): SubscriptionListSummaryResponse => {
+                const subscriptions = response.subscriptions.page || [];
+                const pageInfo = response.subscriptions.pageInfo || {};
+
```

Comment:
Same comment as above

Reply:
It is hard to make a good abstraction here. The prop, in this case, `subscriptions` is different in most situations (processes, productBlocks etc)